### ISSUE TITLE
Logic behind new interface

### ISSUE THREAD

**Issue by hoelzerC** [grimme-lab/dxtb#144]
Arguably the main functionality of `dxtb` will be to calculate 

1. Energies
2. Gradients
3. (Higher derivatives)

To better grasp the design choices behind the new interface (so far docs are not online), I am wondering about the design choices. What is the intended way to calculate energies and forces? 

```
calc = Calculator(numbers, parametrisation, opts)

results = calc.singlepoint(numbers, positions, charges)

gradient = -calc.forces_analytical(numbers, positions, charges).detach()
```
Is that how it should be done? Ideally, a new user will intuitively do this right.
Imo in the upper approach `calc.forces_analytical` should not require a `-1` , as per definition `F = - nabla(U)`.
Also, I am wondering about the content of the `results` object the keys `gradient` and  `total_grad` for instance are not relevant anymore, or are they? And which key unambiguously returns the (total) energy as required by most users?

Suggestions

- Include wrapper function `calc.get_energy(...)` which under the hood returns `calc.singlepoint(...).total.detach().sum()`
- Include wrapper function `calc.get_force(...)` which under the hood returns `-calc.forces_XXX(...)` (whereby `XXX` can be `analytical` per default)

To me, this feels like a more natural approach. Happy to discuss.
